pensieve/config.py

```diff
@@ -250,8 +180,8 @@ class MetricDefinition:
     """
 
     name: str  # implicit in configuration
-    select_expression: str
-    data_source: DataSourceReference
+    select_expression: Optional[str]
+    data_source: Optional[DataSourceReference]
```

Comment:
Hmm -- just to check my understanding, these need be optional because we want to support providing new statistics for default metrics, right?

Reply:
Yep. We'll have to define a default statistic for metrics that already exist in mozanalysis.